compiler/rustc_data_structures/src/steal.rs

```diff
@@ -31,7 +32,7 @@ impl<T> Steal<T> {
 
     pub fn borrow(&self) -> MappedReadGuard<'_, T> {
         ReadGuard::map(self.value.borrow(), |opt| match *opt {
-            None => bug!("attempted to read from stolen value"),
+            None => panic!("attempted to read from stolen value"),
```

Comment:
Is this change related?

Reply:
Yes, `bug` is only defined in rustc_middle: https://doc.rust-lang.org/nightly/nightly-rustc/rustc_middle/macro.bug.html